packages/core/components/DataSourcePrompt/DataSourcePrompt.module.css

```diff
@@ -1,13 +1,40 @@
-.actions-container {
-    align-items: center;
+.checkbox > label > div {
```

Comment:
Just noting that I've added a basic checkbox component with default styling in my open PR that might be useful here

Reply:
Ended up removing this entirely